scripts/ci/libraries/_initialization.sh

```diff
@@ -102,10 +102,10 @@ function initialization::initialize_base_variables() {
     export CURRENT_MYSQL_VERSIONS
 
     # Default Postgres versions
-    export POSTGRES_VERSION=${CURRENT_POSTGRES_VERSIONS[0]}
+    export POSTGRES_VERSION=${POSTGRES_VERSION:=${CURRENT_POSTGRES_VERSIONS[0]}}
 
     # Default MySQL versions
-    export MYSQL_VERSION=${CURRENT_MYSQL_VERSIONS[0]}
+    export MYSQL_VERSION=${MYSQL_VERSION:=${CURRENT_MYSQL_VERSIONS[0]}}
```

Comment:
Hey - would you mind leaving this change out so we can merge https://github.com/apache/airflow/pull/12591 as well?

Reply:
Sure. I want to do it. I just want to test it now for mysql8 as well.

Reply:
Thanks. And I figured, just checking.